GitUI/Editor/FileViewer.cs

```diff
@@ -1434,6 +1508,36 @@ public TestAccessor(FileViewer fileViewer)
             public ToolStripMenuItem CopyToolStripMenuItem => _fileViewer.copyToolStripMenuItem;
 
             public FileViewerInternal FileViewerInternal => _fileViewer.internalFileViewer;
+
+            public IgnoreWhitespaceMethod ignoreWhitespaceMethod
```

Comment:
Incorrect capitalisation
Here and below

Reply:
Thank you for pointing it out. I fixed it.